src/python/pathops/_pathops.pyx

```diff
@@ -669,6 +670,12 @@ cdef class Path:
         )
         cdef Path result = Path.__new__(Path)
         self.path.transform(matrix, &result.path)
+        # TODO: figure out how to transform the original oncurve points using the matrix
+        # result.originalOnCurvePoints = set(
+        #     transform_tuple_point(matrix, pt)
+        #     for pt in self.originalOnCurvePoints
```

Comment:
`SkMatrix` has `mapPoint` and `mapPoints` (for array of SkPoints) methods:
https://api.skia.org/classSkMatrix.html#a545fc5d678f0c07c40636bc4cb699017